backend/api/views/box.py

```diff
@@ -0,0 +1,83 @@
+from boxsdk import Client, OAuth2
+from boxsdk.network.default_network import DefaultNetwork
+from pprint import pformat
+from secret import CLIENT_ID, CLIENT_SECRET, ACCESS_TOKEN
+from StringIO import StringIO
+from boxsdk.exception import BoxAPIException
+
+"""
+Authenticate the user using the client id, secret, and access token.
+### Return box client
+"""
```

Comment:
These docstrings should be placed inside the function that it describes. So for this one it the function signature would be: 

auth_user():
"""comment boi"""